wordpress/user.py

```diff
@@ -17,9 +17,12 @@ def fetch(self, email):
         body = self.client.get_request(f'users/?search={email}')
         if len(body):
             user_id = body[0]['id']
-            body = self.client.get_request(f'users/{user_id}', "context=edit")
-            self._convert_to_resource(body)
+            self.get(user_id)
+        return self
 
+    def get(self, user_id):
```

Comment:
isn't `get` and `fetch` be a bit confusing? what about making `fetch` more specific renaming it to `fetch_by_email`? My point is that I'll have a hard time trying to understand which one of the two to use next time I have to implement something.

Reply:
You are right.